pkg/utils/build/quay.go

```diff
@@ -66,3 +67,25 @@ func DeleteImageRepo(imageName string) (bool, error) {
 	}
 	return true, nil
 }
+
+func DoesTagExistsInQuay(imageURL string) (bool, error) {
```

Comment:
just a very minor thing, but could we add a comment here with the example of how the imageURL looks like?

Reply:
Added now. Thanks.